v2/update_instance_test.go

```diff
@@ -39,6 +51,13 @@ func successUpdateInstanceResponseAsync() *UpdateInstanceResponse {
 	return r
 }
 
+func successUpdateInstanceResponeAsyncWithDashboard() *UpdateInstanceResponse {
+	r := successUpdateInstanceResponseAsync()
+	url := "http://updated.com"
+	r.DashboardURL = &url
```

Comment:
there's a test-scoped `strPtr` method you can use that's a little easier here

Reply:
Thanks for the heads up.